daemon/c_cgroups.c

```diff
@@ -65,7 +65,7 @@
 
 #define CGROUPS_FOLDER MOUNT_CGROUPS_FOLDER
 
-// FIXME: currently replaced by hardware_get_active_cgroups_subsystems() to
+// FIXME: currently replaced by get_active_cgroups_subsystems() to
```

Comment:
if we define this as static list as below here now, we can remove the FIXME and whole comment block here, too

Reply:
done